nucleus/packager/nucleus-grizzly/pom.xml

```diff
@@ -103,6 +103,11 @@
                                     <destFileName>grizzly-npn-bootstrap.jar</destFileName>
                                 </artifactItem>
                                 <!-- Other versioned NPN Bootstrap Jars referenced in new domain.xml (since 5.184) -->
+                                  <artifactItem>
```

Comment:
Is this JAR actually activated anywhere?

Reply:
We agreed with @MeroRai that the JAR file won't be activated automatically and will only be available if users want to use the latest Zulu JDK. They would have to modify JVM options to use this version of the NPN library. This should be documented in the docs.